app/routes/_marketing+/logos/logos.ts

```diff
@@ -112,4 +112,9 @@ export const logos = [
 		alt: 'ESLint',
 		href: 'https://eslint.org',
 	},
+	{
+		src: 'https://svgur.com/i/t9k.svg',
```

Comment:
In the spirit of making sure everything works offline, could you add this svg to the project directly?